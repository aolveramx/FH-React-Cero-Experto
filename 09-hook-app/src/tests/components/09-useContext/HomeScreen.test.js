import {mount} from 'enzyme'
import HomeScreen from "../../../components/09-useContext/HomeScreen";
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Tests in <HomeScreen />', () => {

  const user = {
    name: 'Ari',
    email: 'a@gmail.com'
  }
  
  const wrapper = mount(
    <UserContext.Provider value={{
      user
    }}>
      <HomeScreen />)
    </UserContext.Provider>
  )

  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  });
});