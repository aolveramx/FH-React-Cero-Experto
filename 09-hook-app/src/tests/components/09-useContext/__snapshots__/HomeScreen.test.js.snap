// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HomeScreen /> should render correctly 1`] = `
Array [
  <div>
    <h1>
      Home Screen
    </h1>
    <hr />
    <pre>
      {
   "name": "Ari",
   "email": "a@gmail.com"
}
    </pre>
  </div>,
  ")",
]
`;
